### DISCUSSION TITLE
Separate GitHub Projects

### DISCUSSION THREAD

**Discussion by Pythonic456** [alefragnani/vscode-project-manager#461] [Q&A] [answered]
How can I manage separate GitHub projects with this extension and Source Control in VSCode? I have multiple projects I want to switch between, and one of them to have Source Control for one GitHub repo, and another project to use a different repo, and so on. How can I set this up?

Thanks :)

**Comment by alefragnani** [comment, marked as answer]
Hi @Pythonic456,

Right now, the extension does not check the repo's remote, neither has a way to classify/group the repos.

But, in the upcoming release, you will be able to classify/group projects (issue #50). At first, it will be available only for the **Favorite** projects (the ones you _save_), and after that, it would be added to **Git** and other _auto-detected_ projects as well.

There is also another issue (#407), asking to add GitHub as a new source/auto-detection.

I think a third option could be in the table, with no dependency of the two above. Add a setting to the **Git** panel to _display the repos grouped by remote_. Doing so, would require to check Git info for each repo (a bit of delay) but being _opt-in_ like #378, I think it wouldn't be a problem.

Hope this helps

**Reply by Pythonic456** [reply]
Ok, thanks!
